src/components/Dashboard.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+  <div>
+  <MySidebar></MySidebar>
+</div>
+<div class="main">
+       <div class="card-body text-center">
+    <h4 class="display-4">Configurations Dashboard</h4>
+  </div>
+
+       <table class="table table-striped table-bordered ">
+  <thead class="thead-dark">
+    <tr>
+      <th scope="col"><p align="center">ID</p></th>
+      <th scope="col"><p align="center">Name</p></th>
+      <th scope="col"><p align="center">IP address</p></th>
+      <th scope="col"><p align="center">Port</p></th>
+      <th scope="col"><p align="center">HL7 protocol version</p></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
```

Comment:
I expect that in next ticket/issue table will be populated, in first place with vue-mock-data and later switched to real implementation, done by @nikolaspiric .

So far so good.